src/main/java/seedu/realodex/model/person/Birthday.java

```diff
@@ -0,0 +1,94 @@
+package seedu.realodex.model.person;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.Optional;
+
+/**
+ * Represents a remark in the Realodex
+ */
+public class Birthday {
+
+    public static final String MESSAGE_CONSTRAINTS = "Birthday should be in dd-MMM-yyyy format";
+    public static final DateFormat DATE_FORMAT = DateFormat.getDateInstance(DateFormat.MEDIUM);
+    public static final String INPUT_DATE_PATTERN = "ddMMMyyyy";
+    public final Optional<Date> birthday;
+
+    /**
+     * Constructs a {@code Remark}.
+     *
+     * @param birthday A valid birthday.
+     */
```

Comment:
🆙 🆙 🆙 🆙

Reply:
frick this is wrong shd be {@code Birthday}